pythonforandroid/recipes/python2legacy/patches/ctypes-find-library.patch

```diff
@@ -0,0 +1,28 @@
+diff -ur Python-2.7.2.orig/Lib/ctypes/util.py Python-2.7.2/Lib/ctypes/util.py
+--- Python-2.7.2.orig/Lib/ctypes/util.py	2011-06-11 16:46:24.000000000 +0100
++++ Python-2.7.2/Lib/ctypes/util.py	2015-05-10 15:50:18.906203529 +0100
+@@ -71,7 +71,21 @@
+     def find_library(name):
+         return name
+ 
+-if os.name == "posix" and sys.platform == "darwin":
++# this test is for android specifically shoudl match here and ignore any
++# of the other platform tests below
++if os.name == "posix":
++    def find_library(name):
++        """ hack to find librarys for kivy and android 
++        split the path and get the first parts which will give us 
++        the app path something like /data/data/org.app.foo/"""
++        app_root = os.path.abspath('./').split(os.path.sep)[0:4]
```

Comment:
won't work when deployed on external storage, see 58f7618

Reply:
for what it's worth as I understand it the python2legacy recipe will be broken anyway, and I personally suggest it should be removed. I don't think having the <10% of devices with <21 API support is worth it, giving that problem will only get "better" on its own as time passes. But if we intend to fix the python2legacy recipe this may be worth addressing